sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Keys/src/AzureDataProtectionKeyVaultKeyBuilderExtensions.cs

```diff
@@ -11,7 +11,9 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 
+#pragma warning disable // TODO cleanup of all the warning messages. Issue https://github.com/Azure/azure-sdk-for-net/issues/43768
 namespace Microsoft.AspNetCore.DataProtection
+#pragma warning disable
```

Comment:
Not sure we need two of them.   It's already disabled, we can't disable it more here.  😛